apis/Redis/Redis.Samples.Tests/UpdateInstanceTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Xunit;
+
+[Collection(nameof(RedisFixture))]
+public class UpdateInstanceTest
+{
+    private readonly RedisFixture _fixture;
+    private readonly UpdateInstanceSample _sample;
```

Comment:
Maybe same, don't keep these, just create them when you need them. Particularly the one you are testing should be created inside the test.

Here and for the rest.